frontend/src/App.vue

```diff
@@ -15,7 +15,7 @@
             </main>
         </template>
         <!-- Platform Entry Point -->
-        <template v-else-if="user && !user.password_expired && !termsAndConditionsRequired">
+        <template v-else-if="user && !user.password_expired && !termsAndConditionsRequired && user.email_verified !== false">
```

Comment:
small gripe, but why not have `user.email_verified === true` - it's easier to read?

Reply:
At the point of writing that code I couldn't remember if that was one of the properties that we only set on the user object if it is the 'unexpected' value. Having checked the User view code, I can see that isn't the case, so just setting for `user.email_verified` would be suitable.

For reference, we only set `password_expired` on the User object if the password is expired - otherwise it is undefined. But there we can lean into `undefined === false` so not a problem.

Reply:
Happy to leave it as is - will merge.